### PR TITLE
Bug 1393936: Use hardcoded color values for expanded icons 

### PR DESCRIPTION
**Pull request by gc46**
**Issue:** The expanded icon cannot display clearly in attach to process window after setting the theme to dark

<details>
<summary>Dark Theme</summary>

**Before:**
DarkTheme Before

**After:**
Dark Theme Before

</details>

<details>
<summary>Blue Theme</summary>

**Before**:
Blue Theme Before

**After**:
Blue Theme After
</details>

### CODE REVIEW

**Comment by gc46** [comment]
Note: will remove extraneous comments, but I believe it is better to explain where the hex values come from. This can be polished up.

File: src/SSHDebugPS/UI/ContainerPickerDialogWindow.xaml [hunk 1]
```diff
@@ -30,12 +30,12 @@
         <Style x:Key="GlyphPathStyle"
                TargetType="{x:Type Path}">
             <Setter Property="Stroke"
-                            Value="{DynamicResource {x:Static platformui:TreeViewColors.GlyphBrushKey}}" />
+                            Value="#717171" /> <!-- GlyphBrushKey = Glyph = 717171-->
```
Comment (caslan) [inline src/SSHDebugPS/UI/ContainerPickerDialogWindow.xaml:33]: Ideally we use system colors i.e. something from this enumeration: https://docs.microsoft.com/en-us/dotnet/api/system.windows.systemcolors?view=net-5.0

**Comment by WardenGnaw** [comment]
Can you also share pictures of the window when its on a High Contrast Theme?

**Comment by gc46** [comment]
> Can you also share pictures of the window when its on a High Contrast Theme?

Expanded icon is problematic in High Contrast White.

Screenshots below:

Blue (Extra Contrast):
image

High Contrast Black:
image

High Contrast White:
image

High Contrast 1:
image

High Contrast 2:
image

**Review by graciez — APPROVED** [review]

File: src/SSHDebugPS/UI/ContainerPickerDialogWindow.xaml [hunk 2]
```diff
@@ -30,12 +30,12 @@
...
                TargetType="{x:Type Path}">
             <Setter Property="Stroke"
-                            Value="#717171" /> <!-- GlyphBrushKey = Glyph = 717171-->
+                            Value="{StaticResource {x:Static SystemColors.GrayTextBrushKey}}" />
             <Style.Triggers>
                 <DataTrigger Binding="{Binding  RelativeSource={RelativeSource Mode=FindAncestor, AncestorType={x:Type ListBoxItem}}, Path=IsSelected}"
                              Value="True">
                     <Setter Property="Stroke"
-                            Value="#006CBE" /> <!-- GlyphMouseOverBrushKey = GlyphMouseOver = 006CBE-->
+                            Value="{StaticResource {x:Static SystemColors.HighlightBrushKey}}" />
```
Comment (caslan) [inline src/SSHDebugPS/UI/ContainerPickerDialogWindow.xaml:38]: Did you mean to use HighlightBrushTextKey? Won't this look like blue glyph on blue row highlight?

File: src/SSHDebugPS/UI/ContainerPickerDialogWindow.xaml [hunk 3]
```diff
@@ -30,12 +30,12 @@
         <Style x:Key="GlyphPathStyle"
                TargetType="{x:Type Path}">
             <Setter Property="Stroke"
-                            Value="#717171" /> <!-- GlyphBrushKey = Glyph = 717171-->
+                            Value="{StaticResource {x:Static SystemColors.GrayTextBrushKey}}" />
```
Comment (caslan) [inline src/SSHDebugPS/UI/ContainerPickerDialogWindow.xaml:33]: I think you might want to use one of the control* colors. GrayText is usually for disabled UI elements.

File: src/SSHDebugPS/UI/ContainerPickerDialogWindow.xaml (hunk 2 above)
Comment (graciez) [reply src/SSHDebugPS/UI/ContainerPickerDialogWindow.xaml:38]: right, it doesn't meet the contrast requirements. And this also doesn't work well in high contrast scenarios.

Comment (gc46) [reply src/SSHDebugPS/UI/ContainerPickerDialogWindow.xaml:38]: When using HighlightTextBrushKey, the contrast is not sufficient when High Contrast is off:
image

Blue Theme:
image

High Contrast 1:
image

High Contrast 2:
image

High Contrast Black:
image

High Contrast White:
image

Comment (gc46) [reply src/SSHDebugPS/UI/ContainerPickerDialogWindow.xaml:38]: This is the blue glyph on blue row highlight mentioned above in high contrast off (this is the current implementation):
image

**Pull request merged**